src/types/onyx/IntroSelected.ts

```diff
@@ -1,9 +1,15 @@
-import type {OnboardingPurposeType} from '@src/CONST';
+import type {OnboardingInviteType, OnboardingPurposeType} from '@src/CONST';
 
 /** Model of onboarding */
-type IntroSelected = {
+type IntroSelected = Partial<{
```

Comment:
Why do we need to use partial here?

Reply:
With the new non-organic flow there are cases where the object returned from `NVP_INTRO_SELECTED` is empty, which is why we've changed it to a `Partial` here

Reply:
As you've suggested I've removed the `Partial` here and replaced it with optional parameters for better TS performance